src/python/CMakeLists.txt

```diff
@@ -25,16 +25,9 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 cmake_minimum_required(VERSION 3.18)
-
-message("tritonfrontend python package build skipped when relevant frontends are disabled.")
-message("In order to build tritonfrontend, the following flags are needed: -DTRITON_ENABLE_HTTP=ON -DTRITON_ENABLE_GRPC=ON")
-
 # [DLIS-7232] tritonfrontend package expects all supported packages to be
 # built, without any check/verification for respective frontend enable flags.
 # Support for partial builds(ex: HTTP but not gRPC) will be addressed later.
```

Comment:
> # Support for partial builds(ex: HTTP but not gRPC) will be addressed later.

This is addressed by this PR, right?

Reply:
Thank you for pointing it out. Just removed the comment.